common/graphics/persistent_cache.cc

```diff
@@ -291,17 +293,38 @@ bool PersistentCache::IsValid() const {
   return cache_directory_ && cache_directory_->is_valid();
 }
 
-sk_sp<SkData> PersistentCache::LoadFile(const fml::UniqueFD& dir,
-                                        const std::string& file_name) {
+PersistentCache::SkSLCache PersistentCache::LoadFile(
```

Comment:
Minor nit: Can we make `PersistantCache::SkSLCache` a struct with named members instead of a pair? It's hard to read now because both items are `SkData` and its not clear if the key is first or the value.

Reply:
done